api/METRICS.md

```diff
@@ -0,0 +1,91 @@
+# API proposal for metrics
+
+## Example
+
+```python
+# For most sklearn metrics, we would have their group version that returns a Bunch with fields
+#   * overall: overall metric value
+#   * by_group: a dictionary that maps sensitive feature values to metric values
+
+summary = accuracy_score_by_group(y_true, y_pred, sensitive_features=sf, **other_kwargs)
+
+# Exporting into pd.Series or pd.DataFrame in not too complicated
+
+series = pd.Series({**summary.by_group, 'overall': summary.overall})

```

Comment:
Presumably there's an issue here if one of the groups is called 'overall' but we put that down as a user error?

Reply:
Correct--we let the user handle any exporting and deal with naming conflicts.